code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -689,3 +689,107 @@ GLOBAL_LIST_INIT(ws_survivor_default_loot, list(
 				/obj/item/stack/cable_coil/red,
 				/obj/item/stack/rods/fifty
 				)
+
+/obj/effect/spawner/lootdrop/seeded
+	name = "GO FORTH AND CULTIVATE"
+	icon = 'icons/obj/hydroponics/seeds.dmi'
+	icon_state = "seed"//sneed
+	loot = list(
+		/obj/item/seeds/aloe,
+		/obj/item/seeds/ambrosia,
+		/obj/item/seeds/apple,
+		/obj/item/seeds/cotton,
+		/obj/item/seeds/banana,
+		/obj/item/seeds/berry,
+		/obj/item/seeds/cabbage,
+		/obj/item/seeds/carrot,
+		/obj/item/seeds/cherry,
+		/obj/item/seeds/chanter,
+		/obj/item/seeds/chili,
+		/obj/item/seeds/cocoapod,
+		/obj/item/seeds/coffee,
+		/obj/item/seeds/corn,
+		/obj/item/seeds/eggplant,
+		/obj/item/seeds/garlic,
+		/obj/item/seeds/grape,
+		/obj/item/seeds/grass,
+		/obj/item/seeds/lemon,
+		/obj/item/seeds/lime,
+		/obj/item/seeds/onion,
+		/obj/item/seeds/orange,
+		/obj/item/seeds/peas,
+		/obj/item/seeds/pineapple,
+		/obj/item/seeds/potato,
+		/obj/item/seeds/poppy,
+		/obj/item/seeds/pumpkin,
+		/obj/item/seeds/wheat/rice,
+		/obj/item/seeds/soya,
+		/obj/item/seeds/sugarcane,
+		/obj/item/seeds/sunflower,
+		/obj/item/seeds/tea,
+		/obj/item/seeds/tobacco,
+		/obj/item/seeds/tomato,
+		/obj/item/seeds/tower,
+		/obj/item/seeds/watermelon,
+		/obj/item/seeds/wheat,
+		/obj/item/seeds/whitebeet,
+		/obj/item/seeds/amanita,
+		/obj/item/seeds/glowshroom,
+		/obj/item/seeds/liberty,
+		/obj/item/seeds/nettle,
+		/obj/item/seeds/plump,
+		/obj/item/seeds/reishi,
+		/obj/item/seeds/cannabis,
+		/obj/item/seeds/starthistle,
+		/obj/item/seeds/cherry/bomb,
+		/obj/item/seeds/berry/glow,
+		/obj/item/seeds/sunflower/moonflower)
+	lootcount = 1
```

Comment:
Technically not needed to have this, 1 is already the default lootcount. The same applies to the other lootdrops